airbyte-integrations/connectors/source-pipedrive/source_pipedrive/spec.json

```diff
@@ -0,0 +1,27 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/pipedrive",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Pipedrive Spec",
+    "type": "object",
+    "required": ["api_token", "replication_start_date"],
+    "additionalProperties": false,
+    "properties": {
+      "api_token": {
+        "title": "API Token",
+        "description": "Pipedrive API Token",
+        "airbyte_secret": true,
+        "type": "string"
+      },
+      "replication_start_date": {
+        "title": "Replication Start Date",
+        "description": "UTC date and time in the format 2017-01-25 00:00:00. Any data before this date will not be replicated.",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}$",
```

Comment:
We almost always use the next pattern for date, can we use it with this connector?
`^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$`

Reply:
Yes we can, but this won't match with all the other ts types in this connector

Reply:
That is why I am an adherent of the fact that we translate the date to `Pendulum` and only then work with it and compare.

Reply:
O, I'll do it